byzcoin/bcadmin/cmd_db.go

```diff
@@ -139,6 +143,95 @@ func dbReplay(c *cli.Context) error {
 	return nil
 }
 
+type sumFetcher struct {
+	sum            int
+	bff            byzcoin.BlockFetcherFunc
+	txs            int
+	accepted       int
+	blocks         int
+	timeLastBlock  int64
+	timeLastSum    int64
+	maxTPS         float64
+	maxBlockSize   int
```

Comment:
a max usually involves an upper limit, while here it represents the greatest value found. Maybe `greatestTPS` and `GreatestBlockSize` would be more suitable?

Reply:
Mac dictionary says: `the greatest amount ... recorded` - which fits nicely here...

Reply:
up to you, I was just suggesting a way to be clearer

Reply:
I'll follow the Mac dictionary and keep `max` here ;)